README.md

```diff
@@ -4,12 +4,35 @@ Console program which takes user's query, searches for links and returns results
 
 To install dependencies use command:
 
+1. Install virtual environment: <br />
+```
+pip install virtualenv
+```
+2. Create new environment: <br />
+```
+virtualenv <name_your_environment>
+```
+3. Activate the new environment: <br />
+```
+source <name_your_environment>/bin/activate
+```
+4. Install the requirements in the current environment: <br />
 ```
 pip install -r requirements.txt
 ```
+5. To use program:
 
-To use program:
-
 ```
-    python3 search.py -s "your query" -b "browser name" -o "output file name"
-```
\ No newline at end of file
+python3 search.py -s "your query" -b "browser name" -o "output file name"
+```
+```-s``` - user's searching query. This parameter is <b>mandatory!</b> <br />
+
+```-b``` - browser name:<br />
+* "Chrome" - for Chrome browser <br />
+* "Firefox" - for Firefox browser <br />
+* "Edge" - for Edge browser <br />
+
+Default value is "Chrome". This parameter is <i>optional.</i> <br />
+
+```-o``` - path to <b>.csv</b> file for saving search result. For example: ```"/dir/dir2/result.csv"```
```

Comment:
what  would happened if I put path with unexisting folder?

Reply:
It prints result in console
<img width="880" alt="image" src="https://user-images.githubusercontent.com/15080221/167010180-77b512ad-ac67-400f-82e4-35202bbbcdc2.png">